docs/building-apps/wallet/sep24.mdx

```diff
@@ -238,6 +236,95 @@ val transactions = sep24.getTransactionsForAsset(asset, token)
 
 </CodeExample>
 
+## Submitting Withdrawal Transfer
+
+Previously, we took a look on starting withdrawal flow. Now, let's take a look at full example working with the withdrawal.
+
+First, start the withdrawal:
+
+<CodeExample>
+
+```kt
+val withdrawal = sep24.withdraw(asset, authToken = token)
+```
+
+</CodeExample>
+
+Next, open interactive url as an iframe:
```

Comment:
there are a bunch of options for opening the URL in mobile apps so I'm not sure if we should use the term iframe.

Reply:
What term should we use?

Reply:
Yes, something like that